codefresh.yml

```diff
@@ -15,16 +15,13 @@ steps:
     git: "github"
     stage: "clone"
 
-  install:
-    title: "Install sloctl"
-    type: "freestyle" # Run any command
-    image: "ubuntu:latest" # The image in which command will be executed
-    working_directory: "${{clone}}" # Running command where code cloned
+  version:
+    title: "Get sloctl version"
+    type: "freestyle"
+    image: 'quay.io/codefreshplugins/alpine:3.8'
```

Comment:
this is an old alpine version, any reason why we're using it here? I know it's just for that little wget jq combo, just curious

Reply:
I've just used the alpine version provided by codefresh. I'll upgrade to the newest version served from docker hub

Reply:
sure, I didn't mean it's anything bad to use this one here, was just wondering if there was a deeper meaning here :P